bool load_calibration
bool set_offset
bool enable_calibration_status
bool reset_imu
bool print_calibration_vector
---
bool result